### PR TITLE
Arc 2643 audit log table entry

### PR DESCRIPTION
**Pull request by kamaksheeAtl**
**What's in this PR?**
audit log for deployment flow

### CODE REVIEW

File: src/github/deployment.ts [hunk 1]
```diff
@@ -103,6 +103,7 @@ export const processDeployment = async (
 	const result: DeploymentsResult = await jiraClient.deployment.submit(
 		jiraPayload,
 		webhookPayload.repository.id,
+		webhookPayload.repository.name,
```
[the remark this answers is not in the document]
Comment (gxueatlassian) [reply src/github/deployment.ts:106]: Can you check we can use the full_name instead?

File: src/sync/installation.ts [hunk 2]
```diff
@@ -236,7 +236,7 @@ const sendPayloadToJira = async (task: TaskType, jiraClient, subscription: Subsc
 				});
 				break;
 			case "deployment":
-				await jiraClient.deployment.submit(jiraPayload, repository.id, {
+				await jiraClient.deployment.submit(jiraPayload, repository.id, repository.full_name, {
```
Comment (gxueatlassian) [inline src/sync/installation.ts:239]: I don't this this work. Can you please dig into the api the call the graphql?
I don't think `full_name` appaer in the graphql. Could be nameWithOwner
Try refer to https://docs.github.com/en/graphql/reference/objects#deploymentconnection and check

File: src/github/deployment.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (kamaksheeAtl) [reply src/github/deployment.ts]: yes that would be better

File: src/sync/installation.ts (hunk 2 above)
Comment (gxueatlassian) [reply src/sync/installation.ts:239]: Something like this
image

**Review by gxueatlassian — APPROVED** [review]

**Pull request merged**